src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+
+
+<Note>
+For GUIs and wallet apps, it is recommended to set `announced_channel` to `false` (the default value), as the node is not expected to be reliably online. 
+</Note>
+
+### LightningChannelOptions
+
+
+| Parameter                              | Type    | Description                                                                                   |
+|----------------------------------------|---------|-----------------------------------------------------------------------------------------------|
+| proportional_fee_in_millionths_sats    | integer | Optional, defaults to `0`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| base_fee_msat                          | integer | Optional, defaults to `1000`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| cltv_expiry_delta                      | integer | Optional, defaults to `72`. Blocks until [CheckLockTimeVerify (CLTV)](https://academy.bit2me.com/en/que-es-cltv-bitcoin/) expiry.                                |
+| max_dust_htlc_exposure_msat            | integer | Optional, defaults to `5000000`. Limit our total exposure to in-flight [HTLCs](https://academy.binance.com/en/glossary/hashed-timelock-contract) which are burned to fees as they are too small to claim on-chain.  |
+| force_close_avoidance_max_fee_sats     | integer | Optional, defaults to `1000`. The additional fee we're willing to pay to avoid waiting for the counterparty's locktime to reclaim funds.                            |
+
+
+### LightningClosedChannelsFilter
+
+| Parameter                   | Type    | Description                                                                                                     |
+|-----------------------------|---------|-----------------------------------------------------------------------------------------------------------------|
+| channel_id                  | string  | Optional. Unique string identifying a channel by its ID.                                                        |
+| counterparty_node_id        | string  | Optional. A hexidecimal string identifying a counterparty node. |
+| funding_tx                  | string  | Optional. A transaction ID which added funds.                   |
+| from_funding_value          | integer | Optional. The minimum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| to_funding_value            | integer | Optional. The maximum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| channel_type                | string  | Optional. `Inbound` or `Outbound`. INCONSISTENT WITH `is_outbound` PARAM  |
```

Comment:
You are right, do you prefer that I change it to `"is_outbound": bool`? I guess I used string to save the type as string in the DB but I can easily convert it to boolean in the response.

Reply:
I dont mind which option, though its simpler to wrap head around if consistent. For the end user, it is probably easier to understand the string as surely some will not be knowing that `is_outbound = false` implies it is inbound.
If we are explicit, there is less room for confusion from ambiguity (even when its pretty obvious to us :sweat_smile: )

Reply:
`from_funding_value` and `to_funding_value` in `LightningClosedChannelsFilter` are inconsistent with 
`from_funding_value_sats` and `to_funding_value_sats` in `LightningOpenChannelsFilter`.

Tho it ads to the length of the key, I think including the unit of measure in the key name better clarifies the expected value without user needing to reference docs.

Reply:
DevNote added to track this